packages/ts-utils/src/helpers/typescript-ast.ts

```diff
@@ -46,15 +46,23 @@ export function isTheFirstParentNodeInTheLIne(node: Node): boolean {
 export function findNodeAtPosition(
   sourceFile: SourceFile,
   start: number,
-  length: number
+  length: number,
+  includeJSDocNodes = false
 ): Node | undefined {
   const visitor = (node: Node): Node | undefined => {
     if (isNodeAtPosition(node, start, length)) {
       return node;
     }
 
-    if (node.getStart() <= start && node.getEnd() >= start + length) {
-      return ts.forEachChild(node, visitor);
+    if (includeJSDocNodes) {
```

Comment:
Isn't the JSDocNodes dependent on how the file is parsed?

Reply:
Unfortunately no, they are always not a part of a normal AST tree (node.+children) but instead part of `node.JSDocs`.